profiles/models.py

```diff
@@ -101,6 +111,18 @@ def __init__(self, affiliation_id: str, address: Address, organisation: str, sta
         self.ends = pendulum.timezone('utc').convert(ends) if ends else None
         self.restricted = restricted
 
+    def get_name_list(self) -> List[str]:
+        return [self.department, self.organisation]
+
+    def is_current(self) -> bool:
+        time_now = pendulum.timezone('utc').convert(datetime.now())
+
+        if self.starts <= time_now:
+            if not self.ends or self.ends > time_now:
+                return True
+
+        return False
```

Comment:
Could be:

```python
starts = pendulum.instance(self.starts)
ends = pendulum.instance(self.ends) if self.ends else None

return starts.is_past() and (not ends or ends.is_future())
```

Reply:
`is_past` and `is_future` are handy, less manual work. Changed.